source/TagsWindow.cpp

```diff
@@ -48,38 +51,36 @@ TagsWindow :: TagsWindow(BMessage *fSaveMessage, BHandler *handler)
 	status_t		err;
 	const char		*name;
 
-	BView			*fTagsView;
-	char			bufferTag1[30];
-	char			bufferTag2[30];
-	char			bufferTag3[30];
-
-	// Create the main view
-	fTagsView = new BView(Bounds(), "TagsView", B_FOLLOW_ALL, B_WILL_DRAW | B_FRAME_EVENTS);
-
-	fTag1 = new BTextControl(BRect(10,10,200,30), "tag1", B_TRANSLATE("First Tag: "), NULL, NULL);
-	fTag2 = new BTextControl(BRect(10,40,200,60), "tag2", B_TRANSLATE("Second Tag: "), NULL, NULL);
-	fTag3 = new BTextControl(BRect(10,70,200,90), "tag3", B_TRANSLATE("Third Tag: "), NULL, NULL);
-
-	//Create the OK and UNDO button
-	fCancelButton = new BButton(BRect(210,115,290,135), "cancel",
-			B_TRANSLATE("Cancel"), new BMessage(BUTTON_CANCEL));
-	fOkButton = new BButton(BRect(300,115,380,135), "ok",
-			B_TRANSLATE("OK"), new BMessage(BUTTON_OK));
-
-	/*
-	* Add the view as window's child, set back ground color,
-	* then add the tag TextControl as children
-	*/
-	fTagsView->SetViewColor(ui_color(B_PANEL_BACKGROUND_COLOR));
-
-	AddChild(fTagsView);
-	fTagsView->AddChild(fTag1);
-	fTagsView->AddChild(fTag2);
-	fTagsView->AddChild(fTag3);
-
-	// Add OK and UNDO button as children
-	fTagsView->AddChild(fOkButton);
-	fTagsView->AddChild(fCancelButton);
+	BString tagone = BString();
+	BString tagtwo = BString();
+	BString tagthree = BString();
+
+	fTag1 = new BTextControl("tag1", B_TRANSLATE("First Tag: "), NULL, NULL);
```

Comment:
```suggestion
	fTag1 = new BTextControl("tag1", B_TRANSLATE("First tag:"), NULL, NULL);
```

Same for the following lines.